tox.ini

```diff
@@ -48,9 +48,16 @@ commands =
     {toxinidir}/scripts/ffibuild
 # py310 currently fails with -W error, see: https://gitlab.gnome.org/GNOME/pygobject/-/issues/476
```

Comment:
This gnome issue has been resolved now so we no longer need a separate `py310` run. Might be worth updating that while we're here?

Reply:
Still got an error.